ql/src/test/queries/clientpositive/ctas.q

```diff
@@ -61,11 +61,13 @@ create table nzhang_ctas6 (key string, `to` string);
 insert overwrite table nzhang_ctas6 select key, value from src tablesample (10 rows);
 create table nzhang_ctas7 as select key, `to` from nzhang_ctas6;
 
-
-
-
-
-
-
-
-
+-- ACID CTAS
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.exec.dynamic.partition.mode=nonstrict;
+set hive.stats.autogather=false;
+
+create table acid_ctas_part partitioned by (k)
+  stored as orc TBLPROPERTIES ('transactional'='true')
+  as select key k, value from src order by k limit 5;
+select k, value from acid_ctas_part;
```

Comment:
instead of .q file ..i think adding a replication test with CTAS for partitioned ACID table will be more meaningful.

Reply:
.q file will test the stand-alone CTAS statement. Once we enable true CTAS replication tests as outlined in https://issues.apache.org/jira/browse/HIVE-21677, we will have CTAS replication tests also. In that case, whatever tests we add in this patch will become redundant. So, I feel there's not need to add replication tests explicitly.